features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/ImageTypeResolver.kt

```diff
@@ -9,17 +9,18 @@ package com.datadog.android.sessionreplay.internal.recorder.base64
 import android.graphics.drawable.Drawable
 import android.graphics.drawable.GradientDrawable
 import androidx.annotation.VisibleForTesting
+import com.datadog.android.sessionreplay.internal.recorder.densityNormalized
 
 internal class ImageTypeResolver {
-    fun isDrawablePII(drawable: Drawable, widthInDp: Int, heightInDp: Int): Boolean =
+    fun isDrawablePII(drawable: Drawable, density: Float): Boolean =
         drawable !is GradientDrawable &&
             (
-                widthInDp >= IMAGE_DIMEN_CONSIDERED_PII_IN_DP ||
-                    heightInDp >= IMAGE_DIMEN_CONSIDERED_PII_IN_DP
+                drawable.intrinsicWidth.densityNormalized(density) >= IMAGE_DIMEN_CONSIDERED_PII_IN_DP ||
+                    drawable.intrinsicHeight.densityNormalized(density) >= IMAGE_DIMEN_CONSIDERED_PII_IN_DP
                 )
 
     internal companion object {
-        // material design icon size is up to 48x48
-        @VisibleForTesting internal const val IMAGE_DIMEN_CONSIDERED_PII_IN_DP = 49
+        // material design icon size is up to 48x48, but use 100 to match more images
```

Comment:
I'm curious if we have any explanation why 100 is used?

Reply:
It's an arbitrary value to increase the number of icons that we capture. It's not based on any official guideline, but is an assumption that <100dp images will be non-pii